README.md

```diff
@@ -2,11 +2,12 @@
 ---
 [![Build Status][ti]][tl] [![Apache License][li]][ll]
 
-Muchos supports deploying [Apache Fluo][fluo] and its dependencies to a cluster that can be
-optionally launched in Amazon EC2. Apache Fluo depends on [Apache Accumulo][accumulo], [Apache
-Zookeeper][zookeeper], and [Apache Hadoop][hadoop]. Setting up these dependencies is time consuming.
-Muchos provides a set of helper scripts to automate setting up these dependencies on a cluster. This
-makes it quick for a developer to experiment with Fluo in a realistic environment. 
+**Muchos automates setting up [Apache Accumulo][accumulo] or [Apache Fluo][fluo] (and their dependencies) on a cluster**
+
+Muchos makes it easy to launch a cluster in Amazon's EC2 and deploy Accumulo or Fluo to it. Muchos
+enables developers to experiment with Accumulo or Fluo in a realistic, distributed environment.
+Muchos installs all Apache software using tarball distributions which makes its easy to experiment
```

Comment:
Maybe change `installs all Apache software` to `installs all software` because Muchos also installs non Apache software.